src/ContainerDetails.jsx

```diff
@@ -38,6 +38,14 @@ const ContainerDetails = ({ container }) => (
         <dd>{render_container_state(container)}</dd>
         <dt>{_("Ports")}</dt>
         <dd>{render_container_published_ports(container.Ports)}</dd>
+        <dt>{_("IP Address")}</dt>
+        <dd>{containerDetail ? containerDetail.NetworkSettings.IPAddress : ""}</dd>
+        <dt>{_("IP Prefix Length")}</dt>
+        <dd>{containerDetail ? containerDetail.NetworkSettings.IPPrefixLen : ""}</dd>
```

Comment:
Seems a little unconventional to write it as two separate entities instead of CIDR notation (1.2.3.4/24), but sure.